### PR TITLE
CompCert 64 bit dev version

### PR DESCRIPTION
**Pull request by liyishuai**
Depends on AbsInt/CompCert#364

### CODE REVIEW

File: extra-dev/packages/coq-compcert-64/coq-compcert-64.dev/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+]
+install: [make "install"]
+depends: [
+  "coq" {>= "8.8"}
+  "coq-flocq" {>= "3.2.1"}
+  "coq-menhirlib"
+  "menhir" {>= "20190626"}
+  "ocaml" {>= "4.05.0"}
+]
+synopsis: "The CompCert C compiler (64 bit, using coq-platform supplied version of Flocq)"
```
Comment (silene) [inline extra-dev/packages/coq-compcert-64/coq-compcert-64.dev/opam:32]: Is that right? You do not seem to be patching the sources to use Flocq.

Comment (liyishuai) [reply extra-dev/packages/coq-compcert-64/coq-compcert-64.dev/opam:32]: I've copied a wrong synopsis.

Comment (liyishuai) [reply extra-dev/packages/coq-compcert-64/coq-compcert-64.dev/opam:32]: Strange, I've force-pushed liyishuai@0150711 to my branch but that wasn't reflected in this PR?

Comment (silene) [reply extra-dev/packages/coq-compcert-64/coq-compcert-64.dev/opam:32]: Note that it is not just a matter of changing the synopsis, please also fix the dependencies.

**Pull request merged**